portfolio/src/main/webapp/index.html

```diff
@@ -56,6 +56,16 @@ <h2 class="subheading-text">Comments</h2>
           </form>
         </div>
       </div>
+      <div id="comments-to-show-selector">
+        <form> <!-- not sure if the form wrapper is necessary -->
```

Comment:
what happens if you take it out? :)

Reply:
I did take it out to see, but I forgot to remove the comment. I think it's fine without a form tag, but I'll leave it in to make it more convenient to add more 'filters' for the comment display later on (I can add more selectors to the form, and submit all the values of the form together).